src/reporting/reporting_app/settings/base.py

```diff
@@ -149,6 +144,22 @@ def validate_ldap_settings(server_uri, user_dn_template):
     AUTH_LDAP_GLOBAL_OPTIONS = {ldap.OPT_X_TLS_CACERTFILE: Path(AUTH_LDAP_CERT_FILE)}
 
 
+# The DB settings are defined the same as in the workflow manager
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",  # , 'mysql', 'sqlite3' or 'oracle'.
```

Comment:
does it make sense to make the ENGINE also an environment var?

Reply:
Some of the engines require different "other" configuration items. See [web reflectivity unittest settings](https://code.ornl.gov/reflectometry/web_reflectivity/-/blob/next/src/web_reflectivity/settings/unittest.py) for an example of switching to sqlite.